specification/dimensions/billingaccountname.md

```diff
@@ -0,0 +1,34 @@
+# Billing Account Name
+
+A billing account is a container for resources and/or services that are billed together in an invoice. Common scenarios for using billing accounts include grouping based on organizational constructs and hierarchy-based cost allocation approach.
+
+A billing account name is a display name assigned to a billing account. It is commonly used for cost reporting, invoice reconciliation, and cost allocation scenarios.
```

Comment:
Should we say this is used to differentiate charges from different billing accounts and providers? Cost reporting is too generic, invoice recon isn't completely accurate given this alone doesn't help with that, and allocation is also arguably generic.

Reply:
Sounds good to me. More specific is better.

Curious, as it's defined here, why is invoice recon not accurate? When you get a bill for the usage captured here, thats what our finance / business office folks used to use to reconcile / process payments.